.cloudbuild.yaml

```diff
@@ -97,8 +97,12 @@ steps:
   id: 'buildDeployer'
 - name: 'gcr.io/cloud-builders/docker'
   entrypoint: '/bin/bash'
-  args: ['-c', 'cd /workspace/components/kubeflow/launcher && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
+  args: ['-c', 'cd /workspace/components/kubeflow/tf-launcher && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
   id: 'buildLauncher'
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
```

Comment:
Hi @hougangliu,

This entry is building the new container automatically. Could you please remove this entry? It requires a complicated review process on our side before we can release new containers automatically.

Reply:
thanks, Done